### ISSUE TITLE
Incorrect vertex permulations for hexahedron

### ISSUE THREAD

**Issue by fverdugo** [gridap/Gridap.jl#128]
The vertex permutations seems to be incorrect for the HEX polytope.

In new_master:

```julia
julia> using Gridap.ReferenceFEs

julia> [1,2,3,5,4,6,7,8] in get_vertex_permutations(HEX)
true

julia> [1,2,3,4,5,6,7,8] in get_vertex_permutations(HEX)
false
``` 

That is, the identity permutation its not included in the admissible vertex permutations of the HEX (which is incorrect).

Fixing this is not high priority since this is only needed in 4D or beyond with non oriented faces.

**Comment by fverdugo** [comment]
I have added a temporary fix around line

https://github.com/gridap/Gridap.jl/blob/a27f224673e4748b288b88ec5301d40df7f19026/src/new/ReferenceFEs/ExtrusionPolytopes.jl#L153

This hacky fix has to be removed once the problem is solved

**Comment by fverdugo** [comment]
Fixed by PR #183
